### PR TITLE
Test script install for staging and prod for shell probes

### PR DESCRIPTION
**Pull request by packplusplus**
Shell probe deployments. 

On staging they deploy on pushes to master (or manual)
On prod they deploy on releases (or manual)

### CODE REVIEW

File: .github/workflows/install-prod.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        uses: actions/checkout@v3
+      
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1-node16
+        with:
+          aws-region: ${{ env.AWS_REGION }}
+          aws-access-key-id: ${{ secrets.STAGING_S3_DEPLOYMENT_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.STAGING_S3_DEPLOYMENT_SECRET_KEY }}
+
+      - name: Copy files to the s3 with the AWS CLI
```
Comment (privateducky) [inline .github/workflows/install-prod.yml:33]: shouldn't presto be on this list too?

where are the compiled ones: moonlight and hades?

Comment (packplusplus) [reply .github/workflows/install-prod.yml:33]: They are going to be directory by directory, this is the only one I was done playing with. The rest are coming.

Comment (privateducky) [reply .github/workflows/install-prod.yml:33]: kk

Comment (privateducky) [reply .github/workflows/install-prod.yml:33]: so is this a standalone PR or a WIP?

Comment (packplusplus) [reply .github/workflows/install-prod.yml:33]: stand alone. should be good to go.

**Pull request merged**